### PR TITLE
update for vulnerabile versions

### PR DESCRIPTION
**Pull request by vdonadze**

### CODE REVIEW

**Review by OptimumOpium — CHANGES_REQUESTED** [review]
Please update BOM's version as well

File: gradle.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-release_version=4.2.0
+release_version=4.2.1
```
Comment (OptimumOpium) [inline gradle.properties:1]: I believe we must update the minor version because we update minor versions in dependencies

**Review by OptimumOpium — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 1 above)
Comment (vdonadze) [reply gradle.properties]: done

**Review by OptimumOpium — APPROVED** [review]
Looks good to me. @Nikita-Smirnov-Exactpro Could you please also take a look?

File: build.gradle [hunk 2]
```diff
@@ -3,6 +3,7 @@ plugins {
     id 'maven-publish'
     id "io.github.gradle-nexus.publish-plugin" version "1.0.0"
     id 'signing'
+    id "org.owasp.dependencycheck" version "8.1.2"
```
Comment (Nikita-Smirnov-Exactpro) [inline build.gradle:6]:
```suggestion
    id "org.owasp.dependencycheck" version "8.2.1"
```

**Review by Nikita-Smirnov-Exactpro — DISMISSED** [review]

**Review by OptimumOpium — APPROVED** [review]

**Pull request merged**